cluster/plan.go

```diff
@@ -328,29 +368,31 @@ func (c *Cluster) BuildKubeletProcess(host *hosts.Host, prefixPath string) v3.Pr
 	}
 
 	CommandArgs := map[string]string{
-		"v":                            "2",
-		"address":                      "0.0.0.0",
-		"cadvisor-port":                "0", //depricated in 1.12
-		"read-only-port":               "0",
-		"cluster-domain":               c.ClusterDomain,
-		"pod-infra-container-image":    c.Services.Kubelet.InfraContainerImage,
-		"cgroups-per-qos":              "True",
-		"enforce-node-allocatable":     "",
-		"hostname-override":            host.HostnameOverride,
-		"cluster-dns":                  c.ClusterDNSServer,
-		"network-plugin":               "cni",
-		"cni-conf-dir":                 "/etc/cni/net.d",
-		"cni-bin-dir":                  "/opt/cni/bin",
-		"resolv-conf":                  "/etc/resolv.conf",
-		"allow-privileged":             "true",
-		"cloud-provider":               c.CloudProvider.Name,
-		"kubeconfig":                   pki.GetConfigPath(pki.KubeNodeCertName),
-		"client-ca-file":               pki.GetCertPath(pki.CACertName),
-		"anonymous-auth":               "false",
-		"volume-plugin-dir":            "/var/lib/kubelet/volumeplugins",
-		"fail-swap-on":                 strconv.FormatBool(c.Services.Kubelet.FailSwapOn),
-		"root-dir":                     path.Join(prefixPath, "/var/lib/kubelet"),
-		"authentication-token-webhook": "true",
+		"address":                           "0.0.0.0",
```

Comment:
is it possible to avoid rearranging the fields and do it in a separate PR, and add only the modifications that needed to be done